tests/testsuite/bench.rs

```diff
@@ -1485,3 +1485,81 @@ fn bench_virtual_manifest_all_implied() {
         .with_stdout_contains("test bench_bar ... bench: [..]")
         .run();
 }
+
+#[test]
+fn json_artifact_includes_executable_for_benchmark() {
+    if !is_nightly() {
+        return;
+    }
+
+    let p = project()
+        .file("src/main.rs", "fn main() {}")
```

Comment:
This will have the same non-determinism as the other test.

Reply:
I would not be opposed, I have wanted it in the past.  However, you have to be careful when writing unordered tests because the current code does not do "longest match", so it is easy to create non-deterministic tests (or tests that fail in a confusing way).  Just be careful that one entry is not a prefix of another.

Reply:
Go crazy.  Meanwhile I just trimmed the test.